packages/lesmis-server/src/routes/PDFExportRoute.ts

```diff
@@ -25,67 +20,20 @@ export type PDFExportRequest = Request<
 >;
 
 export class PDFExportRoute extends Route<PDFExportRequest> {
+  private readonly apiClient: TupaiaApiClient;
```

Comment:
This should really be provided by server-boilerplate. Lesmis-server is an Orchestrator Api https://github.com/beyondessential/tupaia/blob/dev/packages/lesmis-server/src/app/createApp.ts#L46

Server-boilerplate already gives us an api-client, but only for the other type of server.
- Microservice Api: https://github.com/beyondessential/tupaia/blob/dev/packages/server-boilerplate/src/microService/api/ApiBuilder.ts#L90 <-- present
- Orchestrator Api: https://github.com/beyondessential/tupaia/blob/dev/packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts <-- missing

Can you please add the same `attachApiClientToContext` method to orchestrator api and then have lesmis-server call it in createApp instead. Then here in this route you can change `this.apiClient.pdfExport()` to `this.req.ctx.services.pdfExport()`.